docs/imviz/plugins.rst

```diff
@@ -429,11 +435,12 @@ catalog dropdown menu.
     This plugin is still under active development. As a result, the search only uses the SDSS DR17 catalog and
     the Gaia catalog and works best when you only have a single image loaded in a viewer.
 
-To load a catalog from a supported `JWST ECSV catalog file <https://jwst-pipeline.readthedocs.io/en/latest/jwst/source_catalog/main.html#output-products>`_, choose "From File...".
-The file must be able to be parsed by `astropy.table.Table.read` and contains the following columns:
+To load a catalog from a supported `JWST ECSV catalog file <https://jwst-pipeline.readthedocs.io/en/latest/jwst/source_catalog/main.html#output-products>`_,
+choose "From File..." from the dropdown menu.
```

Comment:
```suggestion
choose "From File..." from the menu.
```
I vaguely remember that "dropdown" is considered redundant here from the last time I was reading style guidelines.